test/general/implementation_details/indirectly_device_accessible.pass.cpp

```diff
@@ -0,0 +1,461 @@
+// -*- C++ -*-
+//===-- indirectly_device_accessible.pass.cpp -----------------------------------------------===//
```

Comment:
Minor comment. I would recommend don't write file name here(Essentially, we already do it almost everywhere in OneDPL). 
Reason: it duplicates a filename of the file. And such duplication might lead to an issues with filename discrepancy when somebody wants to rename the filename, and rather forget rename this string as well.

Reply:
sure, done